docs/players/creating-bots/agent-builder.mdx

```diff
@@ -26,11 +26,11 @@ You can report issues and submit feature requests through our [Feedback Site](ht
 
 ## Getting Started with Agent Builder
 
-Create a new AI from the Bot Manager and select `Agent Builder` from the list of creation options
-(see the section on [Cloning a Starter Template](./starter-templates) to complete this step).
+Create a new AI from the Bot Manager and select `Agent Builder` from the list of creation options,
```

Comment:
While you're here, we call the button "Create Bot", so it seems odd to talk about it as "Create a new AI":

```suggestion
Create a new Bot from the Bot Manager and select `Agent Builder` from the list of creation options,
```